### ISSUE TITLE
Every time I restart my app, all my transaction history is deleted

### ISSUE THREAD

**Issue by juliobetta** [keymetrics/keymetrics-support#128]
Is this an expected behavior?

**Comment by vmarchaud** [comment]
Hello, 

Yes for now thats the default behavior, PM2 will start to aggregate new transaction from 0 because we thought that when the application restarted it might be because the developer has done a deployment so he want to be sure that the data are corresponding to the deployment.
However it should show you the history of percentiles even across restart (at the top of the transaction panel).

**Comment by juliobetta** [comment]
> ...it might be because the developer has done a deployment...

Not necessarily... maybe the developer wants to compare the time that an specific transaction took before and after the deployment.
